pages/linux/cockpit-ws.md

```diff
@@ -0,0 +1,28 @@
+# cockpit-ws
+
+> Used for communication between the browser application and various configuration tools and services like cockpit-bridge.
+> More information: <https://cockpit-project.org/guide/latest/cockpit-ws.8.html>.
+
+- To check which certificate cockpit-ws will use:
+
+`sudo remotectl certificate`
+
+- Show help options:
```

Comment:
```suggestion
-Display help:
```
This is what we use for help commands across all pages (except some). **Also, move the help command to the last in this page**